app/src/main/java/com/brandon/forzenbook/repository/ForzenRepositoryImpl.kt

```diff
@@ -70,14 +86,16 @@ class ForzenRepositoryImpl(
             Log.e(DATA_ERROR_TAG, "$response")
             Log.e(DATA_ERROR_TAG, "${response.code()}")
             if (response.isSuccessful) {
-                return true
-            }
+                return CreateUserOutcome.CREATE_USER_SUCCESS
+            } else if (response.code() == USER_DUPLICATE_CODE) {
+                return CreateUserOutcome.CREATE_USER_DUPLICATE
```

Comment:
I think throwing as suggested in my other comment may be cleaner, then catching in use case

Reply:
throwing here now and catching in use case, still using the response.code == 409 for the repo.